Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    [Tooltip("The GameObject of the Player's Camera")] [SerializeField] private GameObject camera;
+    [Tooltip("How Fast the Player's Camera moves")] [SerializeField] private float mouseSensitivity = 100f;
+    [Tooltip("How far the Player can reach")] [SerializeField] private float reachDistance = 2.5f;
+    [Tooltip("How Fast the Player Moves")] [SerializeField] private float movementSensitivity = 7f;
+    [Tooltip("Strength of Gravity")] [SerializeField] private float gravity = -9.81f;
+    [Tooltip("The Ground Check Object")] [SerializeField] private Transform groundcheck;
+    [Tooltip("The Radius to check for the ground")] [SerializeField] private float groundDistance = 0.4f;
+    [Tooltip("What layer the ground is located in.")] [SerializeField] private LayerMask groundMask;
+    [Tooltip("The height of a jump")] [SerializeField] private float jumpHeight = 1f;
+    [Tooltip("Audio OneShot to be played durring footsteps.")] [SerializeField] private AudioClip stepAudio = null;
+    [Tooltip("How often a step can be played (In Seconds)")] [SerializeField] private float stepFrequency = 0.68f;
+
+
+    private AudioSource audioSource;
+    private Rigidbody rb;
+    private float lastStep;
+    private bool isPaused = false;
+    private bool isGrounded;
+    private float xRotation = 0f;
+    private CharacterController controller;
+    private InGameMenuController ui;
+    private IInteract lastHeld;
+    private bool isObjHeld;
+
+
+    private Vector3 velocity;
```

Comment:
Very minor, but why the two empty lines between lines 29 and 32?

Reply:
Oh yes, I just put the returns in the wrong place! Thanks for that